nuxeo-features/nuxeo-platform-collections/nuxeo-platform-collections-core/src/test/java/org/nuxeo/ecm/collections/core/test/CollectionAddRemoveTest.java

```diff
@@ -179,6 +179,25 @@ public void testCanAddCollectionNotCollection() {
     }
 
     /**
+     * Tests that we can add a collection with invalid names
```

Comment:
we usually add @since 9.3 in javadoc

Reply:
It's not really important for tests though.

Reply:
This is a backport.  So would we put a @ since 9.3 on code for 8.10?

Reply:
Convention is to keep the since from master branch: we do not edit these on backport cherry picks (if for some specific reason you feel like detailing this, if fix is different on 8.10, for instance, you can, but it's not mandatory)

Reply:
not really important for tests indeed